pages/mesosphere/dcos/2.0/release-notes/2.0.3/index.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: layout.pug
+navigationTitle: Release notes for 2.0.3
+title: Release notes for 2.0.3
+menuWeight: 0
+excerpt: Release notes for DC/OS 2.0.3, including Open Source attribution, and version policy.
+---
+DC/OS&trade; 2.0.3 was released on 19 March 2020.
```

Comment:
I assume this will be updated? :-)

Reply:
Yep good catch. I replaced with ?? for placeholder until date is final.